js/scripts.js

```diff
@@ -1326,22 +1326,22 @@ function plugin_formcreator_addTarget(items_id) {
    });
 }
 
-function plugin_formcreator_deleteTarget(itemtype, target_id) {
-   if(confirm(i18n.textdomain('formcreator').__('Are you sure you want to delete this destination:', 'formcreator'))) {
+$(document).on('click', '.formcreator_delete_target', function() {
```

Comment:
It would be better to keep this as a function, but moving it in plugin_formcreator.